Sources/Misc/DateAndTime/Clock.swift

```diff
@@ -13,21 +13,31 @@
 
 import Foundation
 
-/// A type that can provide the current `Date`
-protocol ClockType: Sendable {
+/// A type that can provide the current `Date` and `DispatchTime`.
+///
+/// - Note: Internal use only
+@_spi(Internal) public protocol ClockType: Sendable {
 
+    /// This property provides the current date as a `Date` object.
     var now: Date { get }
+
+    /// the current time using `DispatchTime`, which is useful for performance measurement or time-sensitive operations
+    /// in GCD contexts.
```

Comment:
The same here. I don't think we need to document these only because we're adding the `@_spi(Internal)` public. As they are basically still internal